src/store/consts/utils.ts

```diff
@@ -563,15 +564,6 @@ export const handleSimulate = async (
       error: errorMessage
     }
   }
-  if (errorMessage.length > 0) {
```

Comment:
are you sure this won't break anything, like cases when error message is needed?

Reply:
yes i'm sure. With this case we receive error even we can swap tokens.